### PR TITLE
Use the new authorizer package

### PR DESCRIPTION
**Pull request by mmmmillar**
There is now a package that contains the authorizer function that we attach to the API Gateway. This PR is to remove the old authorizer and use the package.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -16,7 +16,7 @@ jobs:
 
       - run:
           name: Concatenate all package-lock.json files into single file for checksum
-          command: cat package-lock.json api/package-lock.json authorizer/package-lock.json > combined-package-lock.txt
+          command: cat package-lock.json api/package-lock.json > combined-package-lock.txt
```
Comment (t-davies) [inline .circleci/config.yml:19]: Just interested: does this work just `cat`ing them together (appreciate we were already doing it)? 😕 I would expect the result is invalid JSON.

File: .circleci/config.yml [hunk 2]
```diff
@@ -25,7 +25,8 @@ jobs:
       - run:
           name: Install dependencies
           command: |
-            sudo npm i && pushd api && npm i && popd && pushd authorizer && npm i && popd && pushd jigsaw && npm i && popd
+            sudo npm i --no-cache git
```
Comment (t-davies) [inline .circleci/config.yml:28]: How come this needs `sudo`?

File: package.json [hunk 3]
```diff
@@ -26,11 +29,10 @@
     ]
   },
   "scripts": {
-    "postinstall": "(cd api && echo 'Installing API node modules' && npm i); (cd jigsaw && echo 'Installing jigsaw node modules' && npm i); (cd authorizer && echo 'Installing Authorizer node modules' && npm i);",
+    "postinstall": "(cd api && echo 'Installing API node modules' && npm i); (cd jigsaw && echo 'Installing jigsaw node modules' && npm i);",
```
Comment (t-davies) [inline package.json:32]: Again not related to this change in particular, but have we considered/tried something like [Lerna](https://lerna.js.org/) for dependency management in monorepos? Not that Lerna is perfect, but might avoid some of this dependency install wizardry.

File: .circleci/config.yml (hunk 1 above)
Comment (mmmmillar) [reply .circleci/config.yml:19]: Yeah, the output is just used to generate a checksum which is used as a cache key

Comment (t-davies) [reply .circleci/config.yml:19]: Ah cool yeah I see now, hadn't looked at how it was used. Thanks!

File: .circleci/config.yml (hunk 2 above)
Comment (mmmmillar) [reply .circleci/config.yml:28]: assumption based on the line below it 😄

**Pull request merged**